src/Routers/PagesRouter.js

```diff
@@ -45,6 +45,7 @@ const pages = Object.freeze({
 const pageParams = Object.freeze({
   appName: 'appName',
   appId: 'appId',
+  expiredToken: 'expiredToken',
```

Comment:
Do we really need a new param `expiredToken`? Couldn't we use the existing `token`? I mean across all the changes. I assume the methods would also work for a non-expired token, so the semantic may not even be accurate.